lib/index.js

```diff
@@ -507,14 +527,13 @@ Socket.prototype._flush = function() {
     };
 
     if (flags & zmq.ZMQ_POLLIN) {
-      emitArgs = ['message'];
+      emitArgs = this._read();
 
```

Comment:
if behavior is the same it doesn't matter, i was just quoting [the docs](https://iojs.org/api/stream.html#stream_readable_read_size_1)